db/fixtures/ae_datastore/ManageIQ/Deployment/ContainerProvider/System/StateMachine/Deployment.class/__methods__/provision.rb

```diff
@@ -1 +1,95 @@
+def get_custom_tag(type)
+  type + "_#{$evm.root['automation_task'][:id]}"
+end
 
+def create_custom_tag(name)
+  unless $evm.execute('tag_exists?', 'deploy', "#{name}_#{$evm.root['automation_task'][:id]}")
```

Comment:
@alongoldboim 
Can we do the string interpolation just once and store in a local variable and reuse it in the 3 places
tag_name = "#{name}_#{$evm.root['automation_task'][:id]}"
$evm.execute('tag_create', 'deploy', :name => tag_name, :description => tag_name)